src/features/match/components/MatchDetailMember/MatchMemberList.tsx

```diff
@@ -0,0 +1,176 @@
+import React, {useState} from 'react';
+
+import styled from '@emotion/native';
+import {TouchableOpacity, View} from 'react-native';
+
+import {MatchExitButton} from './MatchExitButton';
+import {MatchMemberListItem} from './MatchMemberListItem';
+import {MatchMemberSettingBottomSheet} from './MatchMemberSettingBottomSheet';
+import {arrowRightXmlData, threeDotXmlData} from '../../../../assets/svg';
+import {Gap} from '../../../../components/Gap';
+import {Icon} from '../../../../components/Icon';
+import {Text} from '../../../../components/Text';
+import {useGetFieldDetail} from '../../hooks/field';
+import {type IUserField, type ITeamEntry} from '../../types';
+
+interface IMatchMemberListProps {
+  id: number;
+  userRole: 'MEMBER' | 'LEADER' | 'GUEST';
+  type: 'MEMBER' | 'DELETE' | 'ASSIGN' | 'REQUEST';
+  members: IUserField[] | ITeamEntry[] | undefined;
```

Comment:
```suggestion
  members?: IUserField[] | ITeamEntry[];
```
사소하긴 하지만 다른 속성처럼 선택적 매개변수로 작성되어도 좋을 것 같습니다.